README.md

```diff
@@ -21,13 +21,12 @@ Add the following dependency to your pom file
 
 ##Usage :
 
-The client needs to be first instantiated with the appropriate application id and key to be able to use
+The client needs to be first instantiated with the appropriate application key to be able to use
```

Comment:
Not related to this change, but this reads weird - can you change to something like:

> First, the client must be instantiated with the appropriate application key (`appKey`)